config/crd/bases/hnc.x-k8s.io_hierarchyconfigurations.yaml

```diff
@@ -112,8 +111,8 @@ spec:
                   description: "Condition contains details for one aspect of the current
                     state of this API Resource. --- This struct is intended for direct
                     use as an array at the field path .status.conditions.  For example,
-                    type FooStatus struct{ // Represents the observations of a foo's
-                    current state. // Known .status.conditions.type are: \"Available\",
+                    \n type FooStatus struct{ // Represents the observations of a
```

Comment:
Any idea what this `\n` is?

Reply:
I think it originates from the upstream K8s `Condition` type doc. It is what controller-gen "spits" out now.

Reply:
That newline just looks weird, it's not showing up anywhere else? Maybe check the comment in the source file to make sure there's no weird newline there? But obviously not a big deal if it's not obvious.

Reply:
This is how the source looks like:

````go
// Condition contains details for one aspect of the current state of this API Resource.
// ---
// This struct is intended for direct use as an array at the field path .status.conditions.  For example,
//
//	type FooStatus struct{
//	    // Represents the observations of a foo's current state.
//	    // Known .status.conditions.type are: "Available", "Progressing", and "Degraded"
//	    // +patchMergeKey=type
//	    // +patchStrategy=merge
//	    // +listType=map
//	    // +listMapKey=type
//	    Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
//
//	    // other fields
//	}
type Condition struct {
````

It seems like the empty line adds `/n` to the CRD field description.